suites/tipset/miner_flow.go

```diff
@@ -1,276 +0,0 @@
-package tipset
```

Comment:
Will add a test similar to this in a follow on. Given the amount that changed in the miner actor, and the amount of :red_circle: :green_circle: in this PR I think this is reasonable.